tests/e2e/virt_backup_restore_suite_test.go

```diff
@@ -0,0 +1,39 @@
+package e2e_test
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/openshift/oadp-operator/tests/e2e/lib"
+	"k8s.io/apimachinery/pkg/util/version"
+)
+
+var _ = Describe("VM backup and restore tests", Ordered, func() {
+	var v *VirtOperator
+	var err error
+
+	var _ = BeforeAll(func() {
+		if !virtTestingEnabled {
+			Skip("Virtualization testing is disabled, skipping tests")
+		}
+
+		v, err = GetVirtOperator(runTimeClientForSuiteRun, kubernetesClientForSuiteRun, dynamicClientForSuiteRun)
+		Expect(err).To(BeNil())
+		Expect(v).ToNot(BeNil())
+
+		minimum, err := version.ParseGeneric("4.13")
+		Expect(err).To(BeNil())
+		if !v.Version.AtLeast(minimum) {
+			Skip("Skipping virtualization testing on cluster version " + v.Version.String() + ", minimum required is " + minimum.String())
+		}
+
+		err = v.EnsureVirtInstallation(5 * time.Minute)
```

Comment:
The errors from many functions underneath are just as the Expect, should this be as well presented using log?

Reply:
The virt_helpers stuff is reasonably well logged, and I was thinking ginkgo logs any error from this Expect nicer than I would anyway. Is there something else that needs to be logged?

Reply:
That's fine. thanks for clarification.